dashboard/prepare_data.py

```
@@ -0,0 +1,335 @@
+
+
+multiple_entry = database_df.query("multiple == True").reset_index(drop=True)
+
+for key in multiple_entry.keys():
+    multiple_entry[key] = multiple_entry[key].apply(get_proper_values_scalar)
+
+values = []
+for key in multiple_entry:
+    try:
+        multiple_entry[key].unique()
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)